tests/conftest.py

```diff
@@ -2014,8 +2017,8 @@ def mcg_job_factory_session(
     )
 
 
-@pytest.fixture(scope='class')
```

Comment:
please make sure that this change from `function` to `class` scope doesn't impact existing test cases